python/hyperon/stdlib.py

```diff
@@ -212,3 +212,52 @@ def load_ascii_atom(space, name):
     return {
         r"load-ascii": loadAtom
     }
+
+def groundedatom_to_python_object(a):
+    obj = a.get_object()
```

Comment:
Not sure if this function is needed with such implementation. You don't need to check if the type is `ValueObject` or `OperationObject`. Both of them have `content` field (and `value` or `op` getter). If there are no other checks, then `groundedatom_to_python_object(a)` is the same as `obj.get_object().content`. The latter is even shorter.